### PR TITLE
Replace magic numbers at main.py

### PR DESCRIPTION
**Pull request by NicholasWon47**
btw i love this module! 

it's quite handy for persuading my boss ;)

### CODE REVIEW

File: perfplot/main.py [hunk 1]
```diff
@@ -247,35 +247,33 @@ def _b(data, kernel, repeat, timer, is_ns_timer, resolution):
     number = 1
     required_timing = 10 * resolution
     min_timing = 0
+
+    # Adapt the number of runs for the next iteration such that the
+    # required_timing is just exceeded. If the required timing and
+    # minimal timing are just equal, `number` remains the same (up
+    # to an allowance of 0.2).
+    allowance = 0.2
```
Comment (nschloe) [inline perfplot/main.py]: Why put this outside the loop?

Comment (NicholasWon47) [reply perfplot/main.py]: I thought it's constant and redefined every iteration. :)

Comment (nschloe) [reply perfplot/main.py]: The gain in "efficiency" does is outweighed by the clarity of the previous setting.

Comment (NicholasWon47) [reply perfplot/main.py]: oh, I agree.

**Pull request merged**